<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchPrintDlg</class>
 <widget class="QDialog" name="BatchPrintDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>批量打印</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>批量打印文件：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtFileName">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnOpenFile">
         <property name="text">
          <string>选择</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>总页数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtTotalPages">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>打印起始页：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtBeginPage">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="inputMask">
          <string>9999</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>终止页：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtEndPage">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="inputMask">
          <string>9999</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPrint">
         <property name="text">
          <string>打印</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tvContent"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
